### DISCUSSION TITLE
CLI - supabase stop --backup output dump.sql erroring on supabase start

### DISCUSSION THREAD

**Discussion by timothybennette8** [supabase/supabase#11439] [Questions]
I'm trying out the new `--backup` flag for `supabase stop` to persist my local project data and out of the box the `dump.sql` file it outputs is giving me an error when I restart supabase.

For testing purposes my process has been:

1.  `supabase start` - with no migration file or `dump.sql'
2.  Create new table in Studio with default row schema (eg. id and date created)
3.  Create row with default data
4.  `supabase db diff -f <arbitrary name>`
5.  `supabase stop --backup`
6.  `supabase start`

On restarting supabase it returns this error:
```
Error: ERROR: schema "pgsodium" does not exist (SQLSTATE 3F000)
At statement 18: --
-- Name: pgsodium; Type: EXTENSION; Schema: -; Owner: -
--

CREATE EXTENSION IF NOT EXISTS "pgsodium" WITH SCHEMA "pgsodium"
``` 
I'm pretty new to SQL and I don't really know how to debug this; but the error is pretty clear - there is no `pgsodium` schema. If I omit the `dump.sql` and start supabase the `pgsodium` schema is present, but I assume that, and all other schema, are being created by the default `seed.sql`? 
If I comment out that statement from `dump.sql` (and subsequent errors) I can see that every time there is a `CREATE EXTENSION IF NOT EXISTS` statement it throws an error due to a missing schema.
Is this the intended behaviour? If so, what I am missing in regards to having the project schema being populated before `dump.sql` is run? Or have I completely got the wrong end of the proverbial stick?

TL; DR
How do I get the sql dump file output by `supabase stop --backup` to successfully populate my project on `supabase start`?
